src/components/BrandConfigProvider/theme/ThemeProvider.tsx

```diff
@@ -1,28 +1,25 @@
 import React from 'react';
+import merge from 'lodash/merge';
+import { RecursivePartial } from '@styles';
 
 import { DefaultTheme as reactNavigationDefault } from '@react-navigation/native';
-
 import {
-  MD3LightTheme as MD3DefaultTheme,
+  DefaultTheme as MD3DefaultTheme,
   useTheme as usePaperTheme,
   adaptNavigationTheme,
   Provider as PaperProvider,
 } from 'react-native-paper';
-import type { MD3Theme } from 'react-native-paper';
-
 import * as baseDefaultTheme from './base/default';
 
-import merge from 'lodash/merge';
-import { RecursivePartial } from '@styles';
-
+// Merges baseDefaultTheme and reactNavigationDefault into MD3DefaultTheme
+// which is the default theme returned by usePaperTheme
 const combinedDefaultTheme = merge(
-  {},
-  reactNavigationDefault,
   MD3DefaultTheme,
+  reactNavigationDefault,
   baseDefaultTheme,
 );
 
-export type Theme = typeof combinedDefaultTheme & MD3Theme;
+export type Theme = typeof combinedDefaultTheme;
```

Comment:
The type of MD3DefaultTheme is MD3Theme which is already included in combinedDefaultTheme so this was redundant.